app/templates/volunteerIndicateInterest.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+
+{% block scripts %}
+{{super()}}
+  <script type="text/javascript" src="{{url_for('static', filename='js/volunteerIndicateInterest.js') }}?u={{lastStaticUpdate}}"></script>
+
+{% endblock %}
+
+{% block app_content %}
+
+<h2>Which programs do you want to receive information about?</h2>
+<input id="{{ user.username }}" hidden>
+  <div class="form-check-inline form-switch">
+  {% for program in programs %}
+    {% set status = '' %}
+      {% if program.id in interests_ids %}
+        {% set status = 'checked' %}
+      {% else %}
+      {% endif %}
+          <input class="form-check-input" type="checkbox" id="{{ program.id }}" {{ status }}>
+          <label class="form-check-label" style="padding-right:50px;" for="{{ program.id }}">{{program.programName}}</label>
+  {% endfor %}
+  </div>
+  <div>
+    <small>
+      <a href="https://www.berea.edu/celts/community-service-programs/volunteer-opportunities/">For more information click here.</a>
+    </small>
+  </div>
+  <div id="flasher-container">
```

Comment:
isn't there a container for the flash messages in base.html?

Reply:
There is this seems to have been created before flashers were added. We have removed it.